src/pages/items/new.svelte

```diff
@@ -27,9 +27,9 @@ let today = new Date()
 
 $: formData.coverage_start_date = today.toISOString().slice(0, 10) //api requires yyyy-mm-dd
 $: formData.purchase_date = formData.coverage_start_date
-$: accountablePersons = $dependents //TODO add current User to this: = [$dependents..., currentUser]
+$: accountablePersons = $dependentsByPolicyId //TODO add current User to this: = [$dependents..., currentUser]
```

Comment:
`$dependentsByPolicyId` isn't actually an array. It's an object with policy-id keys which each have an array as their value.

You may need to do something like this...
```suggestion
$: accountablePersons = $dependentsByPolicyId[$user.policy_id] || [] //TODO add current User to this: = [$dependents..., currentUser]
```

Reply:
nice, everything is back to the level of functioning it was before.